packages/databricks-vscode/src/cli/CliWrapper.ts

```diff
@@ -169,19 +174,37 @@ export class CliWrapper {
 
     @withLogContext(Loggers.Extension)
     public async listProfiles(
-        configfilePath?: string,
+        configfilePath: string,
         @context ctx?: Context
-    ): Promise<Array<ConfigEntry>> {
+    ): Promise<Array<ProfileDetails>> {
+        let stats: Stats;
+        try {
+            stats = await stat(configfilePath);
+            if (
+                stats.mtimeMs <=
+                (this.databricksCfgCache.get(configfilePath)?.lastModified ?? 0)
```

Comment:
Do `mv` or `cp` commands change modification timestamps of the files? And what about windows counterparts?

Reply:
Mv and cp do change the mtime. 

For windows, not sure, but this gets us some performance increase. I will do a more thorough ananlysis in a later PR.